tests/testdata/integration_test_cluster.yaml

```diff
@@ -4,5 +4,6 @@ historyconfig:
   numhistoryshards: 4
   numhistoryhosts: 1
 workerconfig:
+  startworkeranyway: true
```

Comment:
I'm tempted to get rid of this setting and always start a worker, I mean, why not? but you don't have to do that

Reply:
There are so many things that I'm tempted to get rid of :-). Agree, this needs to be done.

Reply:
Yeah but I added this one and feel bad about it